### PR TITLE
feat: Updating some dependencies

### PR DESCRIPTION
**Pull request by YosafatEG**
[CAT-1534](https://jira.expedia.biz/browse/CAT-1534)

Updating some packages:
minimist to ^1.2.8
rotating-file-stream to ^3.1.0
commitizen to ^4.3.0
standard ^17.0.0
tape ^5.6.3
sinon to 15.0.1

### CODE REVIEW

**Comment by kisaiev** [comment]
@YosafatEG 

Changes look good and ready to go, but I noticed that you missed update for Sinon. Was it intentional?

**Comment by YosafatEG** [comment]
> @YosafatEG
> 
> Changes look good and ready to go, but I noticed that you missed update for Sinon. Was it intentional?

Nope, was not intentional, let me add it

File: package.json [hunk 1]
```diff
@@ -55,25 +55,25 @@
   },
   "dependencies": {
     "debug": "^4.1.1",
-    "minimist": "^1.2.0",
+    "minimist": "^1.2.8",
     "pump": "^3.0.0",
-    "rotating-file-stream": "^3.0.0",
+    "rotating-file-stream": "^3.1.0",
     "split2": "^4.0.0",
```
Comment (shellbj) [inline package.json:61]: Unless there is a required security patch, these ranges should remain as is for compatibility.

File: package-lock.json [hunk 2]
```diff
@@ -16443,9 +16409,9 @@
       }
     },
     "minimist": {
-      "version": "1.2.7",
-      "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.7.tgz",
-      "integrity": "sha512-bzfL1YUZsP41gmu/qjrEk0Q6i2ix/cVeAhbCbqH9u3zYutS1cLg00qhrD0M2MVdCcx4Sc0UpP2eBWo9rotpq6g=="
+      "version": "1.2.0",
```
Comment (kisaiev) [inline package-lock.json:16412]: Seems like you downgraded `minimist` instead of updating it to the latest **v1.2.8**.

File: package-lock.json [hunk 3]
```diff
@@ -19559,9 +19450,9 @@
       }
     },
     "rotating-file-stream": {
-      "version": "3.0.4",
-      "resolved": "https://registry.npmjs.org/rotating-file-stream/-/rotating-file-stream-3.0.4.tgz",
-      "integrity": "sha512-Zd+IA5soqBUKtCa7KfMRnF7GKEaRav/dJFU7UHc+r1FMd8VvMLUFE0q1HX5s6XJO1cZ6jAqtTa/ZFnjc7IqlJA=="
+      "version": "3.0.0",
```
Comment (kisaiev) [inline package-lock.json:19453]: Same here. Downgraded `rotating-file-stream` instead of updating it to the latest **v3.1.0**.

**Pull request merged**